staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -397,6 +410,13 @@ type Memory struct {
 	// Defaults to the requested memory in the resources section if not specified.
 	// + optional
 	Guest *resource.Quantity `json:"guest,omitempty"`
+	// SharePages instructs the hypervisor whether to disable shared pages (memory merge, KSM) for this domain.
+	// + optional
+	SharePages *bool `json:"sharepages,omitempty"`
```

Comment:
I'm not 100% sure if you meant to add this to the api but, here's a tip regarding the api violations exceptions in `api/api-rule-violations.list`

The api violation is due to calling the json value `sharepages` instead of the camel cased `sharePages`. The validation is comparing what the json field name is called to what the actual variable name is on the struct.

Reply:
Understood. I will follow the camel case convention and remove the violations.